.gitlab-ci.yml

```diff
@@ -54,7 +54,15 @@ stages:
     - make -j$(grep "core id" /proc/cpuinfo | sort -u | wc -l)
   artifacts:
     paths:
-      - build/
+      - "build/*/*/*/*/*/CMakeCache.txt"
+      - "build/*/*/*/*/*/core/test/"
+      - "build/*/*/*/*/*/cuda/test/"
+      - "build/*/*/*/*/*/omp/test/"
+      - "build/*/*/*/*/*/reference/test/"
+      - "build/*/*/*/*/*/core/libginkgo*"
+      - "build/*/*/*/*/*/cuda/libginkgo*"
+      - "build/*/*/*/*/*/omp/libginkgo*"
+      - "build/*/*/*/*/*/reference/libginkgo*"
```

Comment:
Could you actually add a comment what each folder is representing? Something like (the structure is a complete guess, I don't really know how it looks):
```
# Expected folder structure: build/<cuda_version>/<compiler>/<module>/{debug,release}/{shared,static}/
```
Just in case we change something in the folder structure, so we know right away what to change here.

Reply:
Thanks for the comment, that is done.